src/internal/pjsdb/pjsdb.go

```diff
@@ -0,0 +1,120 @@
+// Package pjsdb exports functions and structs for creating, reading, writing,
+// and deleting Job and Queue objects in PJS.
+package pjsdb
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+)
+
+const (
+	JobErrorCodeFailed       = "failed"
+	JobErrorCodeCanceled     = "canceled"
+	JobErrorCodeDisconnected = "disconnected"
```

Comment:
These appear not to be referred to anywhere. Is GitHub hiding them?

Reply:
These can be removed for now.